components/generic/HeroImage.vue

```diff
@@ -64,11 +109,20 @@
       url: {
         type: String,
         default: ''
+      },
+      compact: {
+        type: Boolean,
+        default: false
       }
     },
+    data() {
+      return {
+        citeCollapsed: true
+      };
+    },
     computed: {
       optimisedImageUrl() {
-        return this.$options.filters.optimisedImageUrl(this.imageUrl, this.imageContentType, { width: 1920 });
+        return this.$options.filters.optimisedImageUrl(this.imageUrl, this.imageContentType, { width: this.exhibition ? 820 : 1920 });
```

Comment:
When would `this.exhibition` ever be `true`?

Consider moving the image width into a prop instead, with the default of 1920.

Reply:
`exhibition` was the previous name for the prop which is now called `compact`, it was changed after Mirjam requested a more generic name, I failed to update this within optimisedImageUrl. We could move the width in to a prop but `compact` is used for conditionally rendering the new header layout as well.